### PR TITLE
Unit test for IPV6 Validation

### PR DESCRIPTION
**Pull request by bipuladh**
Validates that the function used can correctly determine IP Family mismatch.

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.spec.ts [hunk 1]
```diff
@@ -32,6 +32,18 @@ const JSON_CORRECT_MULTIPLE_KEYNAME_VARIATION = [
...
+    kind: 'ConfigMap',
+    data: { maxMonId: '0', data: 'a=10:106:31:93:6789', mapping: {} },
+    name: 'rook-ceph-mon-endpoints',
+  },
+  {
+    kind: 'Secret',
+    data: { userKey: 'AQBV66pedfkUIhAAn/tnB0cvIih5n9DwwxOwBg==', userID: 'csi-cephfs-node' },
+    name: 'rook-csi-cephfs-node',
+  },
+];
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.spec.ts:46]: I am finding it same as `JSON_CORRECT` . Am I missing something ?

**Comment by spadgett** [comment]
/hold
Queue is broken. Hold for #6225

**Pull request merged**